pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -177,6 +177,10 @@ func (plugin *kubenetNetworkPlugin) Name() string {
 	return KubenetPluginName
 }
 
+func (plugin *kubenetNetworkPlugin) Capabilities() uint {
```

Comment:
We could however export NoopNetworkPlugin and then embed that in all the other plugins, and that would get rid of the Capabilities() and Event() methods in them.